### ISSUE TITLE
[web][a11y] Safari mobile enabling a11y

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#71605]
For some versions for Safari mobile (larger than 13.4), enabling a11y does not work due to pointer events vs touch events.
